struts2-bootstrap-showcase/src/main/java/com/jgeppert/struts2/bootstrap/showcase/Index.java

```diff
@@ -33,7 +33,6 @@ public String execute() throws Exception {
         addActionError("A sample Action Error Message!");
         addActionMessage("A sample Action Message.");
         addFieldError("error", "A sample Field Error!");
-        addFieldError("clone", "A sample Field Error!");
```

Comment:
Does it produce any issue? Field errors should be accessed via `.get()` to avoid problems